Source/Core/Core/Config/GraphicsSettings.cpp

```diff
@@ -37,10 +37,10 @@ const ConfigInfo<bool> GFX_LOG_RENDER_TIME_TO_FILE{{System::GFX, "Settings", "Lo
 const ConfigInfo<bool> GFX_OVERLAY_STATS{{System::GFX, "Settings", "OverlayStats"}, false};
 const ConfigInfo<bool> GFX_OVERLAY_PROJ_STATS{{System::GFX, "Settings", "OverlayProjStats"}, false};
 const ConfigInfo<bool> GFX_DUMP_TEXTURES{{System::GFX, "Settings", "DumpTextures"}, false};
-const ConfigInfo<bool> GFX_HIRES_TEXTURES{{System::GFX, "Settings", "HiresTextures"}, false};
-const ConfigInfo<bool> GFX_CONVERT_HIRES_TEXTURES{{System::GFX, "Settings", "ConvertHiresTextures"},
+const ConfigInfo<bool> GFX_CUSTOM_TEXTURES{{System::GFX, "Settings", "HiresTextures"}, false}; // original name retained due to request - #5634
+const ConfigInfo<bool> GFX_CONVERT_CUSTOM_TEXTURES{{System::GFX, "Settings", "ConvertHiresTextures"}, // original name retained due to request - #5634
                                                   false};
-const ConfigInfo<bool> GFX_CACHE_HIRES_TEXTURES{{System::GFX, "Settings", "CacheHiresTextures"},
+const ConfigInfo<bool> GFX_PREFETCH_CUSTOM_TEXTURES{{System::GFX, "Settings", "CacheHiresTextures"}, // original name retained due to request - #5634
```

Comment:
Please remove these comments. Retaining the name is something we try to do for all settings when reasonable, so there's no need to point out that the names of specific settings should be retained.